src/Reactor.jl

```diff
@@ -471,6 +472,53 @@ export getrates
     end
 end
 
+function addreactionratecontributions!(dydt::Q,fragmentbasedrxnarray::Array{W2,2},rxnarray::Array{Int64,2},cs::W,kfs::Z,krevs::Y,massindex::Int64,Mws::Array{Float64,1},fragmentindexes::Array{Int64,1}) where {Q,Z,Y,T,W,W2}
+    (numspcs,numrxns) = size(fragmentbasedrxnarray)
+    half = Int(numspcs/2)
```

Comment:
Can you improve the variable name for `numspcs`?

Reply:
Done